### ISSUE TITLE
Bug OAlab: duplication 'Example' entry in 'File' menu

### ISSUE THREAD

**Issue by revesansparole** [openalea/openalea#203]
In 'File', 'Available projects', the entry 'Examples' looks supiciously duplicated.
See attached pictures
![oalab2](https://cloud.githubusercontent.com/assets/13200498/9137244/b5e24ef6-3d1c-11e5-96c9-4479f242f0dc.png)
![oalab1](https://cloud.githubusercontent.com/assets/13200498/9137245/b5e30814-3d1c-11e5-8ef9-e4850219b133.png)

**Comment by gbaty** [comment]
It seems duplicated but it is not. Indeed, we use name of directory containing project to create menu entry, so if you have projects proj1 and proj2 in
~/openalea/examples and proj3 in ~/Documents/examples, you get two "examples" entry. It is the case here.
However, i agree to say that it seems to be a bug. 

One solution, in case of two directory with same name, is to add parent directory in entry name:
openalea/examples
  - proj1
  - proj2
Documents/examples
  - proj3

An alternative is to merge all project in same entry and separate its:
examples:
  - [openalea]
  - proj1
  - proj2
  - [Documents]
  - proj3

**Comment by revesansparole** [comment]
I haven't created nor installed any particular projects. These two come by default so maybe they need to me renamed differently to avoid confusion.

**Comment by revesansparole** [comment]
I agree with your solution. Down is what they do in PyCharm when two files are open with the same name.
![menu2](https://cloud.githubusercontent.com/assets/13200498/9169117/78e8dad4-3f5a-11e5-9477-e11bac97e273.png)
![menu1](https://cloud.githubusercontent.com/assets/13200498/9169118/78e9b7ba-3f5a-11e5-9688-bfe244de0d55.png)

**Comment by gbaty** [comment]
Thanks for the visual and explicit example.
This is used also in kate text editor (shown like a file browser), eclipse (like pycharm), ...
It's very convenient.

**Comment by gbaty** [comment]
![image](https://cloud.githubusercontent.com/assets/2297918/9278023/212d27b2-42ae-11e5-8a41-1e4ba0621371.png)

First quick fix (waiting for better implementation). I think it is sufficient for the moment.
